control_panel_api/serializers.py

```diff
@@ -161,3 +145,43 @@ def update(self, instance, validated_data):
             )
 
         return super().update(instance, validated_data)
+
+
+class UserUserAppSerializer(serializers.ModelSerializer):
+    """Used from within with UserSerializer to explicitly expose the app
+    but hide the User
+    """
+    app = SimpleAppSerializer()
+
+    class Meta:
+        model = UserApp
+        fields = ('id', 'app', 'is_admin')
+
+
+class UserUserS3BucketSerializer(serializers.ModelSerializer):
```

Comment:
This class name is not obvious

Reply:
Look at the similiar App and S3Bucket stuff how I've done that.  Remember the intention is to explicitly expose certain nested fields, which differ depending on the owning serializer.  

Agree it's not that clear, but I'm not sure what else.  I know @xoen is the naming God so I'd like your input.

Reply:
@jmoz see other comment with rules I used earlier when I was playing with the serializers class names. They seem to make sense to you earlier but it's just an idea.